react/tic-tac-toe/src/app/components/WithLoader/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Spinner from 'react-spinkit';
+
+import styles from './styles.module.scss';
+
+function withLoader(WrappedComponent) {
+  return function Loader({ loader, ...props }) {
+    if (loader) {
```

Comment:
I'd suggest this prop is named `loading` instead. It's a bit clearer. What do you think?